app/helpers/mappings_helper.rb

```diff
@@ -179,7 +179,18 @@ def client_filled_modal
   def mappings_bubble_view_legend
     content_tag(:div, class: 'mappings-bubble-view-legend') do
       mappings_legend_section('Bubble size:', 'The global number of mappings with all other ontologies.', 'mappings-bubble-size-legend') +
```

Comment:
not internationalized

Reply:
Done by me in [8871433](https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/613/commits/88714338ffced784bcdc1d1ab95f3689d7f41e9a)